typescript/audio-summarizer/README.md

```diff
@@ -0,0 +1,188 @@
+# Emojifier 🤖🎨 - A simple audio summarizer with observability on Literal AI
```

Comment:
What about a title like:

```Speech-to-Emoji: Summarize audio with OpenAI Whisper, GPT-4 and Literal AI```

This or the app is the page that we will be sharing on socials, and I think it helps to mention the tools in the title.

On top of that, while I like the sound of Emojifier I am not sure whether it's clear enough what it does by just reading Emojifier... I thought `Speech-to-Emoji` will be more clear and clickbait on socials.

This could be part of a bigger series on marketing Literal AI together with multi-modality:
- speech to text
- speech to speech : voice conversion
- speech to video
- text to speech
- text to image
- text to video
- video to speech

You got it, eventually having a showcase for all 16 combinations of (text, image, speech/audio, video). Can be outsourced to the community, if we create a solid baseline to replicate.

cc @constantinidan @willydouhard

Reply:
We are also missing TypeScript (or Next.js) on the title. Otherwise, Python devs will be disappointed.

Albeit long, a title along the lines of:
```Speech-to-Emoji: TypeScript app to summarize audio with OpenAI Whisper, GPT-4 and Literal AI`

Reply:
On the app, I show that you used `GPT-4o`. Is that important?

If so, and because GPT-4o is trendy, maybe mention it all across the board for more virality.
![Screenshot from 2024-06-18 12-15-03](https://github.com/Chainlit/literal-cookbook/assets/2302748/97c4d8e0-d957-4838-b0a8-7b5440ff2814)

Reply:
I agree that Vercel (Next.js) are more approachable than Microsoft (Typescript) :sweat_smile: 

I converged on : "Speech-to-Emoji: Next.js app to summarize audio with OpenAI Whisper, GPT-4 and Literal AI" and also mentioned gpt-4o instead of gpt-4. (it doesn't have any bearing on the performance on that particular task but you're right that it's trendier)